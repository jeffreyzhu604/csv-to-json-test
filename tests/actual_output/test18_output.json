[
[,
{"field1":"fname","field2":"strength","field3":"slogan.best","field4":"slogan.worst"},,
{"field1":"jeff","field2":100,"field3":" \"whats! good\"","field4":" wh  adbad "},,
{"field2":50,"field3":" whats bad ","field4":"h"},,
]
]
