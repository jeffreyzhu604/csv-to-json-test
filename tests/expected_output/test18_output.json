unknown parameter ../tests/test_files/test_check_col_error18.csv.
[
]
]
